index.html

```diff
@@ -16,12 +16,20 @@
             <a href="#">stephen annor</a>
             <nav class="bars">
                 <i class="fas fa-bars"></i>
+                <div class="mobile-menu" id="mobile-menu">
+                    <img id="btn-cancel" src="images/cancel.png" alt="button-cancel">
+                    <ul>
+                        <li><a href="#portfolio">Portfolio <img src="images/right-arrow.png" alt="arrow-right"></a></li>
+                        <li><a href="#about">About <img src="images/right-arrow.png" alt="arrow-right"></a></li>
+                        <li><a href="#contact">Contact<img src="images/right-arrow.png" alt="arrow-right"></a></li>
+                    </ul>
+                </div>
             </nav>
             <nav class="menu-1">
                 <ul class="menu-2">
                     <li><a href="#portfolio">PORTFOLIO</a></li>
-                    <li><a href="#about">CONTACT</a></li>
-                    <li><a href="#contact">ABOUT</a></li>
+                    <li><a href="#contact">CONTACT</a></li>
+                    <li><a href="#about">ABOUT</a></li>
```

Comment:
- [ ] Be careful, because your links on the desktop version are not working properly and they are not taking us to the proper place, please work on this issue and try to fix it.